Index: maas-1.8.0~alpha1+bzr3486.orig/contrib/maas_local_settings.py
===================================================================
--- maas-1.8.0~alpha1+bzr3486.orig.orig/contrib/maas_local_settings.py	2015-01-23 09:50:03.185579822 -0500
+++ maas-1.8.0~alpha1+bzr3486.orig/contrib/maas_local_settings.py	2015-01-23 09:50:03.181579822 -0500
@@ -14,9 +14,9 @@
     'default': {
         # 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' etc.
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': '',
-        'USER': '',
-        'PASSWORD': '',
+        'NAME': 'maasdb',
+        'USER': 'maas',
+        'PASSWORD': 'maas',
         'HOST': 'localhost',
     }
 }
