Index: maas-0.1+bzr687+dfsg.orig/contrib/maas-http.conf
===================================================================
--- maas-0.1+bzr687+dfsg.orig.orig/contrib/maas-http.conf	2012-06-26 14:13:29.548158883 -0400
+++ maas-0.1+bzr687+dfsg.orig/contrib/maas-http.conf	2012-06-26 14:34:54.388191781 -0400
@@ -2,7 +2,7 @@
 WSGIScriptAlias /MAAS  /usr/share/maas/wsgi.py
 # Preload application when process starts. This will allow publishing
 # the MAAS server existence over Avahi.
-WSGIImportScript /usr/share/maas/wsgi.py process-group=maas
+WSGIImportScript /usr/share/maas/wsgi.py process-group=maas application-group=maas
 WSGIPassAuthorization On
 
 <Directory /usr/share/maas/>
