Index: maas-0.1+bzr1223+dfsg.orig/src/maasserver/api.py
===================================================================
--- maas-0.1+bzr1223+dfsg.orig.orig/src/maasserver/api.py	2012-10-08 13:45:42.932085138 -0400
+++ maas-0.1+bzr1223+dfsg.orig/src/maasserver/api.py	2012-10-08 14:47:34.372180169 -0400
@@ -691,6 +691,14 @@
     'mem': 'memory',
     }
 
+# Map the arch constraint as passed in the request to the name of the
+# arch/subarch in the database.
+_constraints_arch_map = {
+    'i386': 'i386/generic',
+    'amd64': 'amd64/generic',
+    'armhf': 'armhf/highbank',
+    }
+
 
 def extract_constraints(request_params):
     """Extract a dict of node allocation constraints from http parameters.
@@ -704,7 +712,10 @@
     for request_name in _constraints_map:
         if request_name in request_params:
             db_name = _constraints_map[request_name]
-            constraints[db_name] = request_params[request_name]
+            if request_params[request_name] in _constraints_arch_map:
+                constraints[db_name] = _constraints_arch_map[request_params[request_name]]
+            else:
+                constraints[db_name] = request_params[request_name]
     return constraints
 
 
