Index: maas-0.1+bzr740+dfsg.orig/contrib/maas_local_settings.py
===================================================================
--- maas-0.1+bzr740+dfsg.orig.orig/contrib/maas_local_settings.py	2012-07-12 17:15:08.310416293 -0400
+++ maas-0.1+bzr740+dfsg.orig/contrib/maas_local_settings.py	2012-07-12 17:15:08.406416295 -0400
@@ -18,10 +18,10 @@
 MEDIA_ROOT = '/var/lib/maas/media/'
 
 # Use the (libjs-yui) package's files to serve YUI3.
-YUI_LOCATION = '/usr/share/javascript/yui3/'
+#YUI_LOCATION = '/usr/share/javascript/yui3/'
 
 # Use the package's files to serve RaphaelJS.
-RAPHAELJS_LOCATION = '/usr/share/javascript/raphael/'
+#RAPHAELJS_LOCATION = '/usr/share/javascript/raphael/'
 
 # RabbitMQ settings.
 RABBITMQ_HOST = 'localhost'
