Index: maas-0.1+bzr740+dfsg.orig/src/provisioningserver/plugin.py
===================================================================
--- maas-0.1+bzr740+dfsg.orig.orig/src/provisioningserver/plugin.py	2012-07-12 13:51:16.000000000 -0400
+++ maas-0.1+bzr740+dfsg.orig/src/provisioningserver/plugin.py	2012-07-12 14:04:03.614122749 -0400
@@ -12,6 +12,9 @@
 __metaclass__ = type
 __all__ = []
 
+import sys
+sys.path.append('/usr/share/maas')
+
 from getpass import getuser
 
 from formencode import Schema
